{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/forms.py

```diff
@@ -11,9 +13,56 @@ class Meta(admin_forms.UserChangeForm.Meta):
 
 
 class UserCreationForm(admin_forms.UserCreationForm):
+    """
+    Form for User Creation in the Admin Area.  
+    To change user signup, see UserSignupForm and UserSocialSignupForm.
+    """
     class Meta(admin_forms.UserCreationForm.Meta):
         model = User
 
         error_messages = {
             "username": {"unique": _("This username has already been taken.")}
         }
+
+class UserSignupForm(SignupForm):
+    """
+    Form that will be rendered on a user sign up section/screen.
+    Default fields will be added automatically.
+    Check UserSocialSignupForm for accounts created from social.
+    """
+
+    def save(self, request):
+
+        # Ensure you call the parent class's save.
+        # .save() returns a User object.
+        user = super(UserSignupForm, self).save(request)
+
+        # Add your own processing here.
+
+        # You must return the original result.
+        return user
+
+
+    class Meta:
+        model = User
+    
+class UserSocialSignupForm(SocialSignupForm):
+    """
+    Renders the form when user has signed up using social accounts.
+    Default fields will be added automatically.
+    See UserSignupForm otherwise.
+    """
+
+    def save(self, request):
+
+        # Ensure you call the parent class's save.
+        # .save() returns a User object.
+        user = super(UserSocialSignupForm, self).save(request)
+
+        # Add your own processing here.
+
+        # You must return the original result.
+        return user
+
+    class Meta:
+        model = User
```

Comment:
Similar comments here + missing blank line at the end of the file

```suggestion
class UserSocialSignupForm(SocialSignupForm):
    """
    Renders the form when user has signed up using social accounts.
    Default fields will be added automatically.
    See UserSignupForm otherwise.
    """

    class Meta:
        model = User

```